src/CamelKNodeProvider.ts

```diff
@@ -99,7 +99,7 @@ export class CamelKNodeProvider implements vscode.TreeDataProvider<TreeNode> {
 							await this.getIntegrationsFromCamelK().then((output) => {
 								this.processIntegrationList(output);
 							}).catch((error) => { 
-								let errMsg : string = error;
+								let errMsg : string = error as string;
```

Comment:
So this is mostly to avoid the "errMsg.toLowerCase isn't a function" error I see reported sometimes in the Camel K output channel. Though I don't think it's a useful cast, it does remove that problem.